src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int inedex = random.nextInt(Colors.values().length);
+        return Colors.values()[inedex].toString();
```

Comment:
index misspelled, don't use toString - revisit checklist, this issue is mentioned [there](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)